config.toml

```diff
@@ -76,16 +82,16 @@ type  = "sans_serif"
 # Flux hero element
 [params.flux]
 # title    = "Alternative title"
-# subtitle = "The GitOps operator for Kubernetes"
+subtitle = "Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit."
 
 [[params.flux.buttons]]
 text  = "Get started"
 color = "warning"
-url   = "#"
+url   = "https://toolkit.fluxcd.io/get-started/"
```

Comment:
Do these need to be full URLs? If they can be relative, it may help during branch and PR previewing.

Reply:
They can not be relative at this moment as they point towards the Toolkit website.